bin/i18n/resources/dashboard/course_content/sync_in.rb

```diff
@@ -342,7 +342,7 @@ def process
               # We want to make sure to categorize HoC scripts as HoC scripts even if
               # they have a version year, so this ordering is important
               script_i18n_directory =
-                if Unit.unit_in_category?('hoc', script.name)
+                if ScriptConstants.unit_in_category?('hoc', script.name)
```

Comment:
I would write `:hoc` instead of `'hoc'`, which conforms to other places this function is called in our codebase: https://github.com/code-dot-org/code-dot-org/blob/7d740c866a26034cf736821ce3fcadc10cfa49c4/dashboard/lib/certificate_image.rb#L329-L333

And then you wouldn't need the change to `script_constants.rb`